cartographer_grpc/framework/service.cc

```diff
@@ -0,0 +1,10 @@
+#include "cartographer_grpc/framework/server.h"
```

Comment:
This is missing the license header. Same in the other `.cc` file.

Reply:
Done.